app/services/obs-importer.ts

```diff
@@ -230,6 +234,8 @@ export class ObsImporterService extends Service {
         const currentFilePath = path.join(appData, file);
 
         fs.createReadStream(obsFilePath).pipe(fs.createWriteStream(currentFilePath));
```

Comment:
You should remove the old async code.  I'm actually surprised this isn't causing any errors.